server/api/v1/coastlines/coastlines.py

```diff
@@ -0,0 +1,30 @@
+from fastapi import APIRouter, Depends
+from sqlalchemy.ext.asyncio import AsyncSession
+from sqlalchemy.future import select
+from sqlalchemy.orm import joinedload
+
+from server import models, schemas
+# TODO Implement Settings to have hardcoded route prefixes for future use
+from server.core.config import Settings
+from server.db.session import get_db
+
+router = APIRouter(prefix="/api/v1/coastlines", tags=["coastlines"])
```

Comment:
Instead of hardcoding "api/v1/coastlines", let's use those const variables (e.g., `COASTLINE_V1_STR`).

Reply:
I used the COASTLINES_V1_STR, will have to do testing to see if it works.